pages/common/npx.md

```diff
@@ -5,12 +5,20 @@
 
 - Execute the binary from a given npm module:
 
-`npx {{module_name}}`
+`npx {{module_name}} {{command_arguments}}`
```

Comment:
```suggestion
`npx {{module_name}} {{arg1 arg2 ...}}`
```
Why not to use `{{arg1 arg2 ...}}`? ;)

Reply:
`command_arguments` is more generic I think. We should leave this as-is.